src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,104 @@
 import React from 'react';
-
+import {
+Breadcrumb,
+BreadcrumbItem,
+Button,
+Tabs,
+Tab,
+} from 'carbon-components-react';
+const props = {
+tabs: {
+selected: 0,
+triggerHref: '#',
+role: 'navigation',
+},
+tab: {
+href: '#',
+role: 'presentation',
+tabIndex: 0,
+},
+};
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+return (
+<div className="bx--grid bx--grid--full-width landing-page">
+    <div className="bx--row landing-page__banner">
+        <div className="bx--row landing-page__banner">
```

Comment:
Duplicate divs with landing-page__banner class is causing extra padding.

Reply:
duplicate div have been removed, thank you for the support